### ISSUE TITLE
Alpha0.5: Neuropeptide expression bug

### ISSUE THREAD

**Issue by slarson** [openworm/PyOpenWorm#105]
Readme says it should do:
```
  >>> n = P.Neuron()
  >>> n.neuropeptide("TH")
  neuropeptide=`TH'

  >>> s = set(x.name() for x in n.load()) 
  >>> s == set(['CEPDR', 'PDER', 'CEPDL', 'PDEL', 'CEPVR', 'CEPVL'])
  True
```

running it on my system it does this:

```
>>> n = P.Neuron()
>>> n.neuropeptide("TH")
neuropeptide=`TH'
>>> s = set(x.name() for x in n.load())
>>> s == set(['CEPDR', 'PDER', 'CEPDL', 'PDEL', 'CEPVR', 'CEPVL'])
False
```

Note that the final answer is 'false' and should be 'true'

**Comment by travs** [comment]
Fixing #106 means we have added a test that catches this, but it will be an expected failure until the bug is fixed.

**Comment by travs** [comment]
Given this output:
```
In [1]: import PyOpenWorm as P
INFO:rdflib:RDFLib Version: 4.2.0
P.
In [2]: P.connect('default.conf')
INFO:root:Connected to database

In [3]: P.co
P.config      P.configure   P.connect     P.connection

In [3]: n = P.Neuron()

In [4]: n.neurotransmitter('TH')
Out[4]: neurotransmitter=`TH'

In [5]: set(n.load())
Out[5]:
{PDER,
 CEPDR,
 PDEL,
 CEPDL,
 CEPVL,
 PDEL,
 CEPVL,
 PDER,
 CEPVR,
 CEPDR,
 CEPVR,
 CEPDL}
```

It appears that [this](https://github.com/openworm/PyOpenWorm/issues/80#issuecomment-69522839) is still a problem.

**Comment by travs** [comment]
Compare [this line](https://github.com/openworm/PyOpenWorm/blob/5fc9e92c01ccd4a62d6d3f6312ce1094d5dee457/OpenWormData/scripts/insert_worm.py#L163) with sqlite3 output:
```
sqlite> SELECT ID FROM tblentity WHERE Entity = 'Innexin';
355
sqlite> SELECT ID FROM tblentity WHERE Entity = 'Receptor';
361
sqlite> SELECT ID FROM tblentity WHERE Entity = 'Neurotransmitter';
313
sqlite> SELECT ID FROM tblentity WHERE Entity = 'Neuropeptide';
354
```

**Comment by travs** [comment]
Fixed by #115
